hack/bazel-test.sh

```diff
@@ -27,4 +27,4 @@ fi
 bazel test \
     --config=${ARCHITECTURE} \
     --features race \
-    --test_output=errors -- //staging/src/kubevirt.io/client-go/... //pkg/... //cmd/... //tests/framework/...
+    --test_output=errors -- //staging/src/kubevirt.io/client-go/... //pkg/... //cmd/... //tests/framework/... 
```

Comment:
This is added here and removed in the next commit. Leaving it up to you if you want to clean it up properly.

Reply:
If there is something asked from maintainer I will include this fix.